### PR TITLE
Add Darvin Sinaga as an author

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: content/authors/authors.yml [hunk 1]
```diff
@@ -21,3 +21,28 @@
...
+    - name: twitter
+      url: https://twitter.com/perajinkode
+    - name: facebook
+      url: https://www.facebook.com/darvin.sinaga1
+    - name: instagram
+      url: https://www.instagram.com/nagacoder
+    - name: linkedin
+      url: https://www.linkedin.com/in/darvinsinaga/
+    - name: medium
+      url: https://medium.com/@darvinsinaga
```
Comment (zainfathoni) [inline content/authors/authors.yml]:
```suggestion
      url: https://medium.com/@darvinsinaga

```
Tambahkan karakter `newline` di akhir baris

Comment (ghost) [reply content/authors/authors.yml]: `  url: https://medium.com/@darvinsinaga "\n" ` sperti ini?

Comment (zainfathoni) [reply content/authors/authors.yml]: Bukan, seperti yang saya contohkan di _Commit suggestion_ di atas.

Coba deh, buka [file ini](https://github.com/zainfathoni/pejuangkode/pull/7/files#diff-b437a62e3fb23c0c64723e45374de3f1R48), ada _icon_ merah di situ:
image

Untuk keterangan lebih lanjut, coba baca [jawaban StackExchange ini](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file).

File: content/authors/authors.yml [hunk 2]
```diff
@@ -39,3 +39,30 @@
...
+- name: Darvin Sinaga
+  slug: darvin-sinaga
+  bio: |
+    Darvin adalah seorang Frontend Developer Tampan di prospace.io.
+    Saat ini ia sangat tertarik dan menekuni React dan  ekosistemnya.
+    Aktif di beberapa komunitas online maupun offline.
+
+    http://darvinsinaga.com
+  avatar: ./avatars/darvin.jpg
+  featured: true
```
Comment (zainfathoni) [inline content/authors/authors.yml]:
```suggestion
  featured: false
```
Please don't feature yourself before contributing to any article.
`featured` here means that your profile will be put in the home page.

Comment (ghost) [reply content/authors/authors.yml]: oke sure

**Review by zainfathoni — APPROVED** [review]

**Pull request merged**